src/cheesecake.md

```diff
@@ -0,0 +1,52 @@
+# Cheesecake
+
+Sweet dish, pretty creamy, you can use whatever topping you like
+
+![cheesecake](pix/cheesecake.webp)
+
+- Prep time: 25 minutes
+- Cook time: 1hr 10 minutes baking, 2hrs cooling, 6+ hours chilling
+- Serves: 10
+- Difficulty: Medium
+
+## Ingredients
+
+For the crust:
+- 85g normal butter
+- 140g digestive biscuits, crumbled
+- 1tbsp sugar, preferably golden caster
+For the cheesecake:
+- 900g Philadelphia cheese, or any other soft cheese
```

Comment:
```suggestion
- 1tbsp sugar, preferably golden caster

For the cheesecake:
- 900g Philadelphia cheese, or any other soft cheese
```

Without an extra line the `For the cheescake:` line actually ends up on the end of the previous line when rendered.